omero/training-server/playbook.yml

```diff
@@ -32,6 +32,14 @@
         - gcc
         - python-ldap
 
+    - name: Prerequisites for CLI
```

Comment:
:+1: for handling via tasks first. If this becomes handy, a question is whether we would like to have this systematically installed as part of https://github.com/ome/ansible-role-basedeps/blob/master/vars/main.yml? /cc @manics

Reply:
It's not really a basedep in the sense of something that's required for all OMERO systems. It could potentially go into a new role for packages that we want on all our production systems.

Reply:
@manics : do you have any other suggestions for such a role? (And a name?)

Reply:
`ome-ui-tools`, `ome-cli-tools`, `ome-dev-tools`?

Reply:
While migrating roles, I (re-)discovered https://github.com/openmicroscopy/ansible-role-cli-utils, would this be a useful addition there ?

Reply:
looking at https://github.com/openmicroscopy/ansible-role-cli-utils/blob/master/tasks/main.yml
this sounds like a good candidate

Reply:
see https://github.com/openmicroscopy/ansible-role-cli-utils/pull/3